packages/core/src/utils/should-ignore-span.ts

```diff
@@ -0,0 +1,67 @@
+import type { ClientOptions } from '../types-hoist/options';
+import type { SpanJSON } from '../types-hoist/span';
+import { isMatchingPattern, stringMatchesSomePattern } from './string';
+
+/**
+ * Check if a span should be ignored based on the ignoreSpans configuration.
+ */
+export function shouldIgnoreSpan(
+  span: Pick<SpanJSON, 'description' | 'op'>,
+  ignoreSpans: ClientOptions['ignoreSpans'],
+): boolean {
+  if (!ignoreSpans?.length) {
+    return false;
+  }
+
+  if (!span.description) {
+    return false;
+  }
```

Comment:
l: I can't think of an actual example right now but theoretically, shouldn't we still match on the op of a span without a description?

Reply:
decided to leave this as-is for now. We can revisit